TestCases/parallel_regression.py

```diff
@@ -834,7 +834,7 @@ def main():
     Jones_tc.cfg_dir   = "turbomachinery/APU_turbocharger"
     Jones_tc.cfg_file  = "Jones.cfg"
     Jones_tc.test_iter = 5
-    Jones_tc.test_vals = [-5.316335, 0.355081, 44.772280, 2.269966] #last 4 columns
+    Jones_tc.test_vals = [-5.290553, 0.372874, 44.765930, 2.270197] #last 4 columns
```

Comment:
For two other test cases I updated the residuals to have some coverage of the CFL_REDUCTION_TURB option.